Platform/ApollolakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -169,6 +169,10 @@ CONST UINT16 mGpMrbModuleIdOffset[] = {
   0x01B8, 0x01C0, 0x01C8, 0x01D0, 0x01D8, 0x01E0, 0x01E8
 };
 
+CONST UINT16 mUp2ModuleIdOffset[] = {
+  0x00D8, 0x00E0
```

Comment:
Let us use macro NW_GPIO_214/NW_GPIO_215 instead to make it more clear.   This macro is already defined in SiGpioLib.h.  It can be used directly.

Reply:
I understand your comment. Looking through the code, all GpioRead/GpioWrite callers use the same way. I suggest we clean up / refactor GPIO platform code in a separate PR.

Reply:
I am fine if you think it is better to do it in a separate PR.